Assets/MixedRealityToolkit.Providers/ObjectMeshObserver/SpatialObjectMeshObserver.cs

```diff
@@ -331,6 +381,20 @@ private void ApplyUpdatedMeshDisplayOption(SpatialAwarenessMeshDisplayOptions op
             }
         }
 
+        /// <summary>
+        /// 
```

Comment:
I know it's private, but just pointing out the missing comment.

Reply:
already fixed.